src/contracts/Model/Suggestion/SuggestionCollection.php

```diff
@@ -47,4 +56,19 @@ public function truncate(int $count): void
     {
         $this->items = array_slice($this->items, 0, $count);
     }
+
+    public function increaseTotalCount(int $totalCount): void
+    {
+        $this->totalCount += $totalCount;
+    }
+
+    public function decreaseTotalCount(int $totalCount): void
+    {
+        $this->totalCount -= $totalCount;
```

Comment:
perhaps this will not be the case but maybe something should happen when `$this->totalCount` will be lower than 0